addons/analytic/security/analytic_security.xml

```diff
@@ -38,4 +38,12 @@
     </record>
 
 </data>
+<data noupdate="1">
+    <record id="analytic_line_rule_user" model="ir.rule">
+        <field name="name">analytic.line.rule.user</field>
+        <field name="model_id" ref="analytic.model_account_analytic_line"/>
+        <field name="domain_force">[]</field>
+        <field name="groups" eval="[(4, ref('analytic.group_analytic_accounting'))]"/>
+    </record>
+</data>
```

Comment:
why not adding it with the other ir.rules inside that file?

Reply:
Actually, I think we should actually move `account_analytic_line_rule_billing_user` in `analytic` module, but since we are in stable, we should certainly have the same behavior than `account_analytic_line_rule_billing_user` or we should not do that fix to just fix a test in a specific use case.